next.config.js

```diff
@@ -6,5 +6,6 @@ module.exports = {
   env: {
     MAILCHIMP_SUBSCRIBE_URL: process.env.MAILCHIMP_SUBSCRIBE_URL,
     EMAILJS_USER_ID: process.env.EMAILJS_USER_ID,
+    NEXT_PUBLIC_SITE_VERSION: undefined,
```

Comment:
I want this to be defaulted as undefined just to prevent it from accidentally being turned on.